backend/src/core/routes/settings.routes.ts

```diff
@@ -27,10 +27,30 @@ const demoDisplayedValidator = {
   }),
 }
 
+// Only allow updating with versions less than the current package version
+const compareSemverMethod = (value: string, helpers: any) => {
```

Comment:
Should this be less than or equals to?

Reply:
Both work! I made it strictly less than since technically speaking there shouldn't be a case where the `updateAnnouncementVersion` function is called with equal versions, it would mean something had gone wrong. What do you think?